cpp/src/tsne/barnes_hut.cuh

```diff
@@ -264,7 +277,6 @@ void Barnes_Hut(float *VAL, const int *COL, const int *ROW, const int NNZ,
   }
   PRINT_TIMES;
 
-  // Copy final YY into true output Y
```

Comment:
Not necessarily requesting this be reverted, but why the removal of the comment?

Reply:
Re-adding this.